front-end/src/app/reports/report-list/report-list.component.ts

```diff
@@ -107,6 +108,28 @@ export class ReportListComponent extends TableListBaseComponent<Report> implemen
       });
   }
 
+  public confirmUnamend(report: Report): void {
+    this.confirmationService.confirm({
+      message: 'Are you sure you want to unamend this amendent? This action cannot be undone.',
+      header: 'Hang on...',
+      rejectLabel: 'Cancel',
+      rejectIcon: 'none',
+      rejectButtonStyleClass: 'p-button-secondary',
+      acceptLabel: 'Confirm',
+      acceptIcon: 'none',
+      accept: async () => this.unamendReport(report),
+    });
+  }
+
+  async unamendReport(report: Report) {
+    this.itemService
+      .startUnamendment(report)
+      .pipe(take(1), takeUntil(this.destroy$))
+      .subscribe(() => {
+        this.loadTableItems({});
```

Comment:
The ticket references a toast message.  Probably need to add:
```
    this.messageService.add({
      severity: 'success',
      summary: 'Successful',
      detail: 'Report Unamended',
      life: 3000,
    });
```

Reply:
Added in toast.